curator/api/utils.py

```diff
@@ -292,13 +292,27 @@ def prune_closed(client, indices):
     :arg indices: A list of indices to act on
     :rtype: list
     """
+    return prune_open_or_closed(client, indices, append_closed=False)
+
+def prune_opened(client, indices):
+    """
+    Return list of indices that are not open.
+
+    :arg client: The Elasticsearch client connection
+    :arg indices: A list of indices to act on
+    :rtype: list
+    """
+    return prune_open_or_closed(client, indices, append_closed=True)
+
+def prune_open_or_closed(client, indices, append_closed):
```

Comment:
This was a clever approach!  

I'd prefer to see `append_closed` as a keyword argument with a default, e.g. `append_closed=False`, rather than as a positional argument.  Also, I feel that `return_closed` is better semantically than `append_closed`, even if appending is what is happening.  We want the method to _return_ the closed values.